bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZprimeVBF_tau_tau/Zprime_tau_tau_M_1000p0_gl_1_gh_1_kv_1p0_run_card.dat

```diff
@@ -0,0 +1,251 @@
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  Zprime_tau_tau_M_1000p0_gl_1_gh_1_kv_1p0     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100000 = nevents ! Number of unweighted events requested 
+  4488   = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6500.0     = ebeam1  ! beam 1 total energy in GeV
+     6500.0     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+ lhapdf = pdlabel ! PDF set
+# 315200 = lhaid
+ $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number 
```

Comment:
Here CP5 choice is used. Why different from above?

Reply:
@agrohsje They might have 2 different set of gridpacks (one with 16 CUETP and 17/18 CPX) but I think the template and example cards are mixed up.

@aman0404 Can you clarify this?